phpstan.neon

```diff
@@ -10,4 +10,8 @@ parameters:
         - '#Call to an undefined method Facebook\\WebDriver\\WebDriverElement::equals\(\)#'
         - '#Unsafe usage of new static\(\)#'
 
+        # Parameter is intentionally not part of signature to not break BC
```

Comment:
TODO: v2 clean the error

Reply:
This is checked implicitly - once the root issue is resolved (param is part of signature), phpstan will trigger an error if the ignored error is no longer matched.